distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -1828,17 +1830,14 @@ public void testDefaultValuesCreate() throws Exception {
                                 .isPresent());
                 return null;
             });
-            getServiceManager().stopFeature(true, "catalog-core-validationparser");
         }
     }
 
     @Test
     public void testDefaultValuesUpdate() throws Exception {
-        getServiceManager().startFeature(true, "catalog-core-validationparser");
-
         final String customMetacardTypeName = "custom";
         File file = ingestDefinitionJsonWithWaitCondition("defaults.json", () -> {
-            expect("Service to be available: " + MetacardType.class.getName()).within(10,
+            expect("Service to be available: " + MetacardType.class.getName()).within(30,
```

Comment:
Are there cases where this could actually take longer than 10 seconds for the metacard type to get registered?